core/src/com/unciv/ui/utils/KeyPressDispatcher.kt

```diff
@@ -63,15 +63,16 @@ data class KeyCharAndCode(val char: Char, val code: Int) {
         /** Guaranteed to be ignored by [KeyPressDispatcher.set] and never to be generated for an actual event, used as fallback to ensure no action is taken */
         val UNKNOWN = KeyCharAndCode(Input.Keys.UNKNOWN)
 
-        // Kludges because we got crashes: java.lang.NoSuchMethodError: 'int kotlin.CharCodeKt.access$getCode$p(char)'  
-        fun Char.toCode() =
-            try { code } catch (ex: Throwable) { null }
-                ?: try { @Suppress("DEPRECATION") toInt() } catch (ex: Throwable) { null }
-                ?: 0
-        fun Int.makeChar() =
-            try { Char(this) } catch (ex: Throwable) { null }
-                ?: try { toChar() } catch (ex: Throwable) { null }
-                ?: Char.MIN_VALUE
+        // Kludges because we got crashes: java.lang.NoSuchMethodError: 'int kotlin.CharCodeKt.access$getCode$p(char)'
+        //TODO fixed by removing UncivServer from the desktop module - clean up comments and all uses
+        fun Char.toCode() = code
+//            try { code } catch (ex: Throwable) { null }
+//                ?: try { @Suppress("DEPRECATION") toInt() } catch (ex: Throwable) { null }
+//                ?: 0
+        fun Int.makeChar() = Char(this)
+//            try { Char(this) } catch (ex: Throwable) { null }
+//                ?: try { toChar() } catch (ex: Throwable) { null }
+//                ?: Char.MIN_VALUE
```

Comment:
This change is causing a crash, cannot start any game at all now:
```
Exception in thread "Thread-12" java.lang.NoSuchMethodError: kotlin.CharCodeKt.access$getCode$p(C)I
	at com.unciv.ui.utils.KeyCharAndCode$Companion.makeChar(KeyPressDispatcher.kt:72)
	at com.unciv.ui.utils.KeyCharAndCode$Companion.ctrl(KeyPressDispatcher.kt:78)
	at com.unciv.ui.worldscreen.WorldScreen.addKeyboardPresses(WorldScreen.kt:277)
	at com.unciv.ui.worldscreen.WorldScreen.<init>(WorldScreen.kt:186)
```
It was actually expected, the comment above has warned about it.
> // Kludges because we got crashes: java.lang.NoSuchMethodError: 'int kotlin.CharCodeKt.access$getCode$p(char)'

Reply:
Did you Build->Clean and Invalidate Caches?

Reply:
Huh... `Build -> Clean project` does not help, but `File -> Invalidate caches` has done the job. 👍🏻